homeassistant/components/sensor/bitcoin.py

```diff
@@ -0,0 +1,238 @@
+"""
+homeassistant.components.sensor.bitcoin
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Bitcoin information service that uses blockchain.info and its online wallet.
+
+Configuration:
+
+You need to enable the API access for your online wallet to get the balance.
+To do that log in and move to 'Account Setting', choose 'IP Restrictions', and
+check 'Enable Api Access'. You will get an email message from blockchain.info
+where you must authorize the API access.
+
+To use the Bitcoin sensor you will need to add something like the
+following to your config/configuration.yaml
+
+sensor:
+  platform: bitcoin
+  wallet: 'YOUR WALLET_ID'
+  password: YOUR_ACCOUNT_PASSWORD
+  currency: YOUR CURRENCY
+  display_options:
+    - type: 'exchangerate'
```

Comment:
Out of curiosity, why make this a list of dictionaries with only one key? Why not just make this a list and have it look something like this:

```yaml
sensor:
  display_options:
    - timestamp
    - blocks_size
    - total_btc
```